extension/js/common/composer.ts

```diff
@@ -513,10 +513,6 @@ export class Composer {
           // not a draft - updating draft that was since sent as a message (in another window), and is not a draft anymore
           this.urlParams.draftId = ''; // forget there was a draftId - next step will create a new draftId
           await this.draftSave(true); // forceSave=true to not skip
-        } else if (!this.urlParams.draftId && Api.err.isNotFound(e)) {
```

Comment:
You still need this. New messages (not a reply) may also be opened from drafts.

Reply:
Okay, got it